scripts/configure-hypervisor.sh

```diff
@@ -354,6 +351,11 @@ generate_qemu_options() {
 		qemu_options+=(size:--disable-dmg)
 		qemu_options+=(size:--disable-parallels)
 		qemu_options+=(size:--disable-vxhs)
+
+		if [[ "${qemu_version_minor}" -ge 1 ]]; then
```

Comment:
this won't work for versions >= 4 with a minor equals to 0, for example `4.0.0`.

I think the expression we want here is `[[ "${qemu_version_major}" -ge 4 || ( "${qemu_version_major}" -eq 3  &&  "${qemu_version_minor}" -ge 1 ) ]]`

Reply:
Yes, would be better